.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: "Feature request"
+description: "You have a neat idea that should be implemented?"
+labels: ["enhancement", "0. to triage"]
+body:
+  - type: markdown
+      attributes:
+          value: |
+              ### 👍Thanks for reporting issues back to Nextcloud!
+              This is the issue tracker of Nextcloud, if you have any support question please check out [support](https://nextcloud.com/support)
+              To make it possible for us to help you please fill out below information carefully.
+
+              If you have any questions, head over to the [forums](https://help.nextcloud.com/c/apps/mail).
+              Note that Nextcloud is an open source project backed by Nextcloud GmbH. Most of our volunteers are home users and thus primarily care about issues that affect home users. Our paid engineers prioritize issues of our customers. If you are neither a home user nor a customer, consider paying somebody to fix your issue, do it yourself or become a customer.
```

Comment:
we didn't have it before, let's not add this now either :+1: 

```suggestion
```

Reply:
yes, removed, but on our old one, there were only 2 sections, i added 2 more, like in contacts and calendar. Is that alright?